hack/ci/ci-bootstrap.yaml

```diff
@@ -65,10 +64,13 @@ files:
           CI_URL="gs://kubernetes-release-dev/ci/$CI_VERSION-bazel/bin/linux/amd64"
 
           for CI_PACKAGE in "${PACKAGES_TO_TEST[@]}"; do
-            echo "* downloading package: $CI_URL/$CI_PACKAGE.$PACKAGE_EXT"
-            ${GSUTIL} cp "$CI_URL/$CI_PACKAGE.$PACKAGE_EXT" "$CI_DIR/$CI_PACKAGE.$PACKAGE_EXT"
-            ${SUDO} dpkg -i "$CI_DIR/$CI_PACKAGE.$PACKAGE_EXT" || echo "* ignoring expected 'dpkg -i' result"
+            echo "* downloading binary: $CI_URL/$CI_PACKAGE"
+            ${GSUTIL} cp "$CI_URL/$CI_PACKAGE" "$CI_DIR/$CI_PACKAGE"
+            chmod +x "$CI_DIR/$CI_PACKAGE"
+            mv "$CI_DIR/$CI_PACKAGE" "/usr/bin/$CI_PACKAGE"
           done
+
+          systemctl restart kubelet
```

Comment:
as mentioned here:
https://github.com/kubernetes-sigs/cluster-api-provider-gcp/pull/284/files#r377658995

this might not be complete fix, in case the base OS is missing some deps.